woocommerce/woocommerce
# E2E: Remove WP Latest-2 from release tests

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR removes "WP Latest-2" from the list of environments being tested during release tests. WooCommerce now only supports up to WP Latest-1 so we need to make this appropriate adjustment to our release tests.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to [Smoke test release](https://github.com/woocommerce/woocommerce/actions/workflows/smoke-test-release.yml).
2. Select this branch ( `e2e/remove-wp-l-2` ).
1. Select any recent valid release version.
2. Run the workflow. Wait for it to finish.
3. In the list of jobs, verify that you see the job "Test against WP Latest-1". There should be no "Test against WP Latest-2" job.
    <img width="224" alt="buQLejuy4C" src="https://github.com/woocommerce/woocommerce/assets/4509348/67d33754-fc11-4d7e-a8ab-3bf1cfa42851">
4. Click on the "Test against WP Latest-1" job.
1. Look for the step `Launch WP Env` and expand it.
2. Expand the `Run pnpm run env:test` command to see the environment variables passed onto it.
3. Verify that the value for the `WP_ENV_CORE` variable is `WordPress/WordPress#6.2.2`. The number after the `#` character should be the correct WP L-1 version, currently 6.2.2.
    <img width="592" alt="Iy9difP5ia" src="https://github.com/woocommerce/woocommerce/assets/4509348/3ac5e7ab-e3cc-4763-8d3c-421a03d63eae">
1. In the Slack notification for release tests, verify that the `WP Latest-1` section shows the correct L-1 version. Also verify that there's no `WP Latest-2` section.
    <img width="290" alt="OOQFdgMfdA" src="https://github.com/woocommerce/woocommerce/assets/4509348/c5c8c173-3f82-45d6-964d-a5774626c8c5">





### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
